docroot/sites/all/modules/custom/ppgetstat/ppgetstat.module

```diff
@@ -651,33 +708,19 @@ function _ppgetstat_api_call($site, $endpoint, $vars, $page = 0) {
  */
 function _ppgetstat_sync_user_list_step($variables, $page, &$context) {
   for ($i = 0; $i < 10; $i++) {
-    $r = _ppgetstat_api_call(
-      'https://www.drupal.org/',
-      'api-d7/user.json',
-      $variables,
-      $page
-    );
+    $r = _ppgetstat_api_call('api-d7/user.json', $variables, $page);
```

Comment:
Please, dont delete first parameter from _ppgetstat_api_call function.
In addition to the drupal.org we also have association api at the another domain
Also we will work to get localize.drupal.org api live and scan them too.